### DISCUSSION TITLE
Do you have patreon or something?

### DISCUSSION THREAD

**Discussion by jiriks74** [inkstitch/inkstitch#1075] [General]
Hello,
I love your work, you saved me a lot of money with your program, it's just awesome. And since I think you do it in your free time and I think you should get at least some support from people.
I'm trying to translate it to my naitive language and I'd really like to send you at least a tip for your work. Do you have patreon?

**Comment by lexelby** [comment]
Thanks!  I'm glad you like Ink/Stitch :)

I haven't set up a patreon or tip jar yet because I just don't need money for Ink/Stitch, to be honest. @kaalleen, if you or the other frequent contributors wanted to set one up, that would be fine with me.  Maybe you need a new embroidery machine or supplies?

I have a nice machine that does what I need.  I have an etsy shop that gives me enough cash for stabilizer and thread.  If my machine breaks down, then maybe it will be time to set up a tip jar :)

I don't want people to feel like they have to pay for Ink/Stitch.  It's free for a reason.  I wanted to get into embroidery and didn't want to pay a ton of money for software.  And now that I'm hooked on it, I want others to be able to "break into" embroidery without having to pay hundreds or even thousands for software.

Thank you so much for working on translation.  Your time is valuable and it makes me happy that you want to spend some of it on our project.

**Reply by jiriks74** [reply]
@lexelby 
> I don't want people to feel like they have to pay for Ink/Stitch. It's free for a reason. 

I understand what you mean. I don't think that patreon isn't meaning that people have to pay, for me it's just a way to show I value your time and like your product. I want to give you something for it, even if it's just a little. I don't want to push you into making one, I'd contribute with code, if I was good in coding, but I'm not on that level, yet. Maybe one day.

Thanks again. I'll learn my mom to use it and it'll be our way to go :)

**Reply by kaalleen** [reply]
This is really a kind offer. And teaching your mom is really the best way to go ;)
It is so great to see how the Ink/Stitch community grows.

I just wanted to link this discussion to this thread over here #953 Where we have thought about a donation option to support macOS users better by getting a developer license to sign our releases.
